docs/DeveloperGuide.md

```diff
@@ -170,6 +170,33 @@ sequence diagram shows the interactions within `Model` when editing a person.
 
 ![EditSequenceDiagram-Model](images/EditSequenceDiagram-Model.png)
 
+### Delete Tag Command
```

Comment:
I think the heading should be renamed to "Delete Tag Feature" instead? On that note, I think it is better to shift it such that it is right after "Add Tag Feature" so that it is more coherent?